nym-credential-proxy/nym-credential-proxy/src/http/router/api/v1/ticketbook/shares.rs

```diff
@@ -100,50 +100,51 @@ pub(crate) async fn query_for_shares_by_id(
     let uuid = random_uuid();
```

Comment:
Hot take: uuid are too long to print in tracing statement. I prefer to truncate at 8 characters